src/items/equipment/honorbringer.json

```diff
@@ -111,7 +112,7 @@
ne day rebel, he demanded the complete eradication of all nonVesk societies on the planet. Eshovok personally led his armies, and the sight of their emperor cleaving through entire formations with Honorbringer filled his vesk soldiers with fervor, inspiring unstoppable battle frenzies in many. After achieving victory, he often personally executed the enemy commanders who had most distinguished themselves in combat, considering this death a sign of heartfelt respect for noble opponents. Since the Veskarium’s foundation, Honorbringer has passed from emperor to emperor, regularly serving as a traditional weapon, cruel scepter, or metaphorical flag symbolically planted in the ground before an emperor would declare a newly conquered world annexed. It’s currently kept in the Imperial Palace on Vesk Prime.</p>\n<p>Due to this adamantine doshko’s unusual design, it functions as a special weapon for proficiency purposes. In addition, Honorbringer can perform lethal swings, allowing its user to use a soldier’s kill shot class feature with the weapon once per hour as if they were a 20th-level soldier. A soldier with the kill shot class feature who wields this weapon can instead use the kill shot ability with Honorbringer once per minute without spending a Resolve Point. Finally, the weapon grants its wielder the inspired frenzy ability, below.</p>\n<p>It’s rumored that if a Veskarium emperor uses Honorbringer to disgrace Damoritosh, the weapon would shatter into many pieces, reforming only once a new emperor has taken the throne. Permanently destroying the weapon might require giving one shard to a resurrected member of each species the Veskarium has conquered, or perhaps the weapon would disintegrate once the Veskarium is destroyed forever.</p>\n<p><strong>Inspired Frenzy (Ex)</strong> As a swift action after you have used Honorbringer to kill at least three enemies in the last minute, you can inspire a battle frenzy in any number of your willing allies within 60 feet, including yourself. While in a battle frenzy, a creature gains 20 temporary Hit Points, a +4 bonus to melee damage rolls, a +2 bonus to saving throws, and a –2 penalty to AC. It also can’t use any abilities that require patience or concentration, such as spellcasting, or any Charisma-, Dexterity-, or Intelligence-based skills except Acrobatics, Intimidate, and Piloting. The only reactions a creature in a battle frenzy can take are attacks of opportunity. While you wield Honorbringer and are affected by the battle frenzy, you also gain the frightful presence (Alien Archive154) universal creature ability.</p>\n<p>The frenzy lasts for a number of rounds equal to 3 × your key ability score modifier, though an affected creature can end its own battle frenzy as a move action. After you use this battle frenzy, you can’t do so again until after you rest 10 minutes to regain Stamina Points.</p>\n<p><em>Enable/Disable&nbsp;</em><span><em>Inspired Frenzy modifiers in the Modifiers tab.</em></span></p>"
```

Comment:
```suggestion
      "value": "<h2>&gt;@UUID[Compendium.sfrpg.rules.P7TMxsRPHGxzlHmh]{Artifact}</h2><p>This unique, six-bladed, doshko-like weapon is said to have been bestowed upon Eshovok the Farsighted by Damoritoshto aid the emperor in his conquest of the Vesk home world. Eshovok sought to conquer what is now called Vesk Prime, and rather than subjugate enemies who might one day rebel, he demanded the complete eradication of all non-Vesk societies on the planet. Eshovok personally led his armies, and the sight of their emperor cleaving through entire formations with Honorbringer filled his vesk soldiers with fervor, inspiring unstoppable battle frenzies in many. After achieving victory, he often personally executed the enemy commanders who had most distinguished themselves in combat, considering this death a sign of heartfelt respect for noble opponents. Since the Veskarium’s foundation, Honorbringer has passed from emperor to emperor, regularly serving as a traditional weapon, cruel scepter, or metaphorical flag symbolically planted in the ground before an emperor would declare a newly conquered world annexed. It’s currently kept in the Imperial Palace on Vesk Prime.</p>\n<p>Due to this adamantine doshko’s unusual design, it functions as a special weapon for proficiency purposes. In addition, Honorbringer can perform lethal swings, allowing its user to use a soldier’s kill shot class feature with the weapon once per hour as if they were a 20th-level soldier. A soldier with the kill shot class feature who wields this weapon can instead use the kill shot ability with Honorbringer once per minute without spending a Resolve Point. Finally, the weapon grants its wielder the inspired frenzy ability, below.</p>\n<p>It’s rumored that if a Veskarium emperor uses Honorbringer to disgrace Damoritosh, the weapon would shatter into many pieces, reforming only once a new emperor has taken the throne. Permanently destroying the weapon might require giving one shard to a resurrected member of each species the Veskarium has conquered, or perhaps the weapon would disintegrate once the Veskarium is destroyed forever.</p>\n<p><strong>Inspired Frenzy (Ex)</strong> As a swift action after you have used Honorbringer to kill at least three enemies in the last minute, you can inspire a battle frenzy in any number of your willing allies within 60 feet, including yourself. While in a battle frenzy, a creature gains 20 temporary Hit Points, a +4 bonus to melee damage rolls, a +2 bonus to saving throws, and a –2 penalty to AC. It also can’t use any abilities that require patience or concentration, such as spellcasting, or any Charisma-, Dexterity-, or Intelligence-based skills except Acrobatics, Intimidate, and Piloting. The only reactions a creature in a battle frenzy can take are attacks of opportunity. While you wield Honorbringer and are affected by the battle frenzy, you also gain the frightful presence (Alien Archive154) universal creature ability.</p>\n<p>The frenzy lasts for a number of rounds equal to 3 × your key ability score modifier, though an affected creature can end its own battle frenzy as a move action. After you use this battle frenzy, you can’t do so again until after you rest 10 minutes to regain Stamina Points.</p>\n<p><em>Enable/Disable Inspired Frenzy modifiers in the Modifiers tab.</em></p>"
```